_includes/cloud-query/cloud-query-choose-db.md

```diff
@@ -1,6 +1,6 @@
-There are two methods for defining the database you want to query:
+There are two methods for defining the database you want to query in the UI:
 
 |Method | Instructions  |
 | --- | ----------- |
 | Database Dropdown | Click the dropdown in the top right labeled **Database** and select your database. This will default to the database most recently created |
-| Query alias | You can ignore the dropdown selection and direct any query to a particular database by aliasing/prepending the query with `“{<database name>}”`. example: `{my_db}select 1`  |
\ No newline at end of file
+| Query alias | You can bypass the dropdown database selection and direct any query to a particular database by aliasing/prepending the query with `“{<database name>}”`. example: `{my_db}SELECT 1`  |
```

Comment:
You can leave out "You can bypass the dropdown database selection and" because it's implied by the structure of the table.

First method to choose a db is to use the dropdown
Second method is to "prepend/alias" the database in the query.

You don't need to state what's being ignored.